test-requirements.txt

```diff
@@ -1,11 +1,7 @@
-six
-pyOpenSSL
-flake8
-flake8-mutable
-flake8-print
-pylint
-setuptools-lint
+# General Deps
```

Comment:
I'm reworking this a bit, not only is specifying the exact version of ansible not really needed for the existing set of tests (it applies to the molecule testing), but even for the molecule testing, I will do a separate pip install within the `commands` section.